addons/base_import_module/controllers/main.py

```diff
@@ -15,21 +15,17 @@ def wrap(*args, **kw):
 class ImportModule(Controller):
 
     def check_user(self, uid=None):
-        if uid is None:
-            uid = request.uid
```

Comment:
May be we should keep this code, if uid is None then we will consider uid = request.env.user.id